templates/index.html

```diff
@@ -10,6 +10,7 @@
       <p id="tooltip-single" class="tooltip"><span class="key">ENTER</span>&nbsp;will&nbsp;submit, <span class="key">SHIFT</span>&nbsp;+&nbsp;<span class="key">ENTER</span>&nbsp;will&nbsp;create&nbsp;a&nbsp;newline</p>
       <p id="tooltip-multi" class="tooltip"><span class="key">CTRL</span>&nbsp;+&nbsp;<span class="key">ENTER</span>&nbsp;will&nbsp;submit, <span class="key">ENTER</span>&nbsp;will&nbsp;create&nbsp;a&nbsp;newline</p>
       <a id="btn-submit" class="animated fadeIn">ENCRYPT AND SHARE</a>
+      <p id="info" class="warn">Read "How it works" before use to make sure your secret is safe!</p>
```

Comment:
Could the `"How it works"` be linked to the help?

Reply:
Good point ;)